pshtt.py

```diff
@@ -10,13 +11,19 @@
 import os
 import utils
 import logging
-import requests_cache
 
-from models import Domain, Endpoint
+try:
+    from urllib import parse as urlparse # Python 3
+except ImportError:
+    import urlparse # Python 2
```

Comment:
Even though this tool currently requires Python 2, I made the new `urlparse` import agnostic so that it wouldn't be a barrier if this tool can eliminate its Python 2 dependency.